packages/gatsby/cache-dir/api-runner-ssr.js

```diff
@@ -6,10 +6,45 @@
 
 const apis = require(`./api-ssr-docs`)
 
-module.exports = (api, args, defaultReturn) => {
+/**
+ * Some apis should only be implemented once. Given a list of plugins, and an
+ * `api` to check, this will return [] for apis that are implemented 1 or 0 times.
+ *
+ * For apis that have been implemented multiple times, return an array of paths
+ * pointing to the file implementing `api`.
+ *
+ * @param {array} pluginList
+ * @param {string} api
+ */
+const duplicatedApis = (pluginList, api) => {
+  let implementsApi = []
+
+  pluginList.forEach(p => {
+    if (p.plugin[api]) implementsApi.push(p.path)
+  })
+
+  if (implementsApi.length < 2) return [] // no dupes
+  return implementsApi // paths to dupes
+}
+
+// Run the specified api in any plugins that have implemented it
+const apiRunner = ({ api, args, defaultReturn, checkDupes = false }) => {
```

Comment:
I like named arguments (should have used them here originally) but we'd need to update the node/browser api runners as well to keep things consistent internally. So could you revert this change for now and perhaps do the named args change in another PR?

Reply:
👍